### PR TITLE
Added 2 files for the integration of the Google Maps API

### PR DESCRIPTION
**Pull request by epartakki**
- Check html and js files
- Had issues with coordinates
- I am not sure my function makes sense (the one with the API)

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class User < ApplicationRecord
+  has_secure_password
```
Comment (sunniraleigh) [inline app/models/user.rb:2]: I'm not what this means? What is it related to?

**Review by kvangreen — COMMENTED** [review]
Looks good!

**Review by sunniraleigh — COMMENTED** [review]
Great work @epartakki ! Please look at my comment about `has_secure_password`

**Review by amarrap — APPROVED** [review]
Great!

**Pull request merged**